### PR TITLE
PROJECT: DESIGN YOUR OWN GRID-BASED FRAMEWORK

### PR DESCRIPTION
**Pull request by bolah2009**
# PROJECT: DESIGN YOUR OWN GRID-BASED FRAMEWORK

This is the eight project of the Main HTML/CSS curriculum at [Microverse](https://www.microverse.org/) - @microverseinc

* The objective is to create a css grid based framework.

#### How to use

The Project use the media query (break-point) with  `min-width: 400px`, `min-width: 700px` and `min-width 900px` and the prefix -s, -m and -l was used for the class names respectiely.

##### Small Screen 
Live ScreenShot for minimum screen width of 0px

### Medium Screen
Live ScreenShot for minimum screen width of 768px

### Large Screen
Live ScreenShot for minimum screen width of 992px

#### Authors

* [@Torres-ssf](https://github.com/Torres-ssf)
* [@bolah2009](https://github.com/bolah2009/)

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+          <img class="html-badge-img" alt="HTML and CSS Badge" src="assets/badge-html-css.svg">
+        </div>
+
+        <h1 class="page-heading">HTML and CSS</h1>
+
+        <h2 class="page-sub-heading">Project: Design Your Own Grid-Based Framework</h2>
+
+        <div class="row">
+
+          <nav class="left-navbar none column-3-l">
```
Comment (AdnanAfsari) [inline docs/index.html]: you can not use nav tag more then one in a single  project so nav tag for main navigation and use div for other navigation.

Comment (bolah2009) [reply docs/index.html]: Thanks for the observation @adnanafsari. According to HTML5 spec is not inappropriate, so please can you enlighten us on why it shouldn't be used? Thanks.

**Comment by bolah2009** [comment]
> Hi Guys @bolah2009 @Torres-ssf
> Lots of hard work huh?Your project looks perfect and you did a good job with your code but there few things you have to change, please do and submit for another review.
> 
> * as project assignment you have to create another repo only for your css grid-framework then import the framework in your project and build the project with it in a new repo.
> * live demo  is not working, please update it with this link
>   https://raw.githack.com/bolah2009/CSS-Grid-Based-Framework/dev/docs/index.html

Hey @AdnanAfsari, thanks for the review :+1: 

1. The instruction is to use a single repository for the framework and the page using it. Please, check the [assignment page](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49037275/)

> Use a single repository for this project and make sure to include a folder with an example of a page using the grid framework
2. The project is on the `/docs` folder and will be available for live version as soon as the PR is merged. The GitHub pages have been set to `/docs` on the master branch.

Please reconsider my points and get back to us on what to do next.
Thanks again!

File: docs/index.html (hunk 1 above)
Comment (AdnanAfsari) [reply docs/index.html]: hay its not inappropriate or wrong but its best practice because of the screen reader as i explained when we were talking.

**Comment by AdnanAfsari** [comment]
Hi well done guys. Your project looks perfect 👍 
You can go ahead and merge the branch.

- sorry i was checking odin project requirement but as microverse requirement project structure is totally fine.thanks for your patient.

Happy coding :)

**Pull request merged**